gamestonk_terminal/stocks/technical_analysis/ta_controller.py

```diff
@@ -174,19 +169,141 @@ def call_quit(self, _):
         """Process recom command"""
-        tradingview_view.print_recommendation(other_args, self.ticker)
+
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="recom",
+            description="""
+            Print tradingview recommendation based on technical indicators.
+            [Source: https://pypi.org/project/tradingview-ta/]
+        """,
+        )
+        parser.add_argument(
+            "-s",
+            "--screener",
+            action="store",
+            dest="screener",
+            type=str,
+            default="america",
+            choices=["crypto", "forex", "cfd"],
+            help="Screener. See https://python-tradingview-ta.readthedocs.io/en/latest/usage.html",
+        )
+        parser.add_argument(
+            "-e",
+            "--exchange",
+            action="store",
+            dest="exchange",
+            type=str,
+            default="",
+            help="""Set exchange. For Forex use: 'FX_IDC', and for crypto use 'TVC'.
+            See https://python-tradingview-ta.readthedocs.io/en/latest/usage.html.
+            By default Alpha Vantage tries to get this data from the ticker. """,
+        )
+        parser.add_argument(
+            "-i",
+            "--interval",
+            action="store",
+            dest="interval",
+            type=str,
+            default="",
+            choices=["1M", "1W", "1d", "4h", "1h", "15m", "5m", "1m"],
+            help="""Interval, that corresponds to the recommendation given by tradingview based on technical indicators.
+            See https://python-tradingview-ta.readthedocs.io/en/latest/usage.html""",
+        )
+
+        try:
+            ns_parser = parse_known_args_and_warn(parser, other_args)
+            if not ns_parser:
+                return
+            tradingview_view.print_recommendation(
+                ticker=self.ticker,
+                screener=ns_parser.screener,
+                exchange=ns_parser.exchange,
+                interval=ns_parser.interval,
+            )
+
+        except Exception as e:
+            print(e, "\n")
 
     def call_pr(self, other_args: List[str]):
         """Process pr command"""
-        finnhub_view.pattern_recognition_view(other_args, self.ticker)
+
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="pr",
+            description="""
+            Display pattern recognition signals on the data. [Source: https://finnhub.io]
+        """,
+        )
+        parser.add_argument(
+            "-r",
+            "--resolution",
+            action="store",
+            dest="resolution",
+            type=str,
+            default="D",
+            choices=["1", "5", "15", "30", "60", "D", "W", "M"],
+            help="Plot resolution to look for pattern signals",
+        )
+
```

Comment:
Can we add export?

Reply:
think you commented this twice on this command.  If you meant to comment on the `summary` command then that is a string output, so not exportable based on our conversation in disc.